.env.default

```
@@ -1,26 +1,29 @@
 
-AZURE_MISTRAL_ENDPOINT=https://Mistral-small-alkemio-serverless.swedencentral.inference.ai.azure.com
-AZURE_MISTRAL_API_KEY=<your-mistral-ai-key>
+EMBEDDINGS_API_KEY=<your-embeddings-api-key>
+EMBEDDINGS_ENDPOINT=https://api.scaleway.ai/v1
+EMBEDDINGS_MODEL_NAME=qwen3-embedding-8b
 
-LLM_DEPLOYMENT_NAME=deploy-gpt-35-turbo
-EMBEDDINGS_DEPLOYMENT_NAME=embedding
 RABBITMQ_HOST=rabbitmq
 RABBITMQ_USER=alkemio-admin
 RABBITMQ_PASSWORD=alkemio!
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RABBITMQ_PASSWORD key should go before the RABBITMQ_USER key

(UnorderedKey)